gradle.properties

```diff
@@ -18,4 +18,6 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+kapt.incremental.apt=true
+MyMoviesApp_ApiKey="c4bafe4c76558f68a89e13760b0255dd"
```

Comment:
The aim of configuring the `API key` is to ensure that is not visible in the repository. Ensure that users on Github can't see this.

Reply:
I would suggest you create a file called `apikey.properties` on the root project folder then add this line into the file `API_KEY=XXXXXXXXXXX` where the `XXXXXXXXXXX` is you actual apiKey. To avoid these keys showing up in the repository, make sure to exclude this file from being commited in by adding to your `.gitignore` file.

Then add these lines in the `app/build.gradle` so the apiKey can be accessed from the project.
```
def apikeyPropertiesFile = rootProject.file("apikey.properties")
def apikeyProperties = new Properties()
apikeyProperties.load(new FileInputStream(apikeyPropertiesFile))

```
 Then still in the `app/build.gradle` under the section `android` and `defaultConfig` the line I have marked as add `// ADD THIS LINE`  on the snippet below.
```
android {

  defaultConfig {

    buildConfigField("String", "API_KEY", apikeyProperties['API_KEY']) // ADD THIS LINE
   
  }
}
```

Now the ApiKey can be accessed anywhere in the code as below
`val apiKey = BuildConfig.API_KEY`

Once you are done with this I would like you to document the same on the README so that the other team members can add the the `apikey.properties` in there local project add their own apiKeys.